falco/templates/pod-template.tpl

```diff
@@ -134,6 +134,10 @@ spec:
           {{- end }}
       {{- end }}
       volumeMounts:
+        - mountPath: /etc/falco
+          name: rulesfiles-install-dir
+        - mountPath: /usr/share/falco/plugins
+          name: plugins-install-dir
```

Comment:
Unless I'm missing something...
Should those paths be mounted only when falcoctl's install or follow features are enabled? :thinking: